jsonnet/components/node-exporter.libsonnet

```diff
@@ -23,6 +24,65 @@ function(params)
       },
     },
 
+    // This changes node-exporter to be scraped with validating TLS.
+    serviceMonitor+: {
+      metadata+: {
+        labels+: {
+          'monitoring.openshift.io/scrape-profile': 'full',
+        },
+      },
+    },
+
+    minimalServiceMonitor: generateServiceMonitor.minimal(
```

Comment:
@JoaoBraveCoding Is this the metrics list exposed by node-exporter for `scrapeProfile: minimal` setting, same for other components

Reply:
This metric list only concerns metrics exported by `node-exporter` that are then used in alerts, rules, telemetry or console.
If you look at the metric list of another component you will see completely different sent of metrics.
Let me know if that answers the question or if I misunderstood it.

Reply:
yes, that's my question, thanks for the explanation